ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.cpp

```diff
@@ -159,29 +161,42 @@ void BalovnevModelPlugin::computeForces(double vertical_cut_depth)
     + d*a3 * (2*s + 4*ls*tan(delta))*(d*g*sd/2 + c*(1/tan(phi))
     + g*q + BURIED * (d-ls*sin(beta)) * g*sd * (1-sin(phi))/(1+sin(phi)));
 
+  // NOTE: With accurate depths being fed to the equation, the forces are now
+  //       far too large, and immediately deflect the scoop off course. Until we
+  //       find the parameter that's causing this, we have added a fudge factor
+  //       that lowers both forces by a factor of 1/100.
+  constexpr double FUDGE_FACTOR = 1.0; // 0.05
```

Comment:
What is the `0.05` inline comment?

Reply:
Removed along with the long-winded explanation. FUDGE_FACTOR is enough explanation on its own. I'll keep this variable for now, but hope to remove it by the time the epic branch is merged.